scripts/travis_install

```diff
@@ -42,10 +41,15 @@ EOF
 )
 
 conda install --yes $DEPS_TEST
-conda install --yes abstract-rendering
-conda install --yes multiuserblazeserver pillow
-##hack - we need latest blaze right now
-conda install --yes -c blaze blaze
+
+conda install --yes nodejs
+echo "node version $(node -v)"
+echo "npm version $(npm -v)"
+
+if [[ "$TRAVIS_PYTHON_VERSION" == '2.7' ]]; then
+    conda install --yes pdiff
+    conda install --yes boto
+fi
```

Comment:
Can this be integrated in the devenv script? I would really like to have the ultimate way to install all dependencies, like `devenv --I-want-them-all`. However, this PR close to being strictly about dev, so it's not a blocker.